sansio_lsp_client/events.py

```diff
@@ -65,6 +90,34 @@ class LogMessage(ServerNotification):
     message: str
 
 
+class WorkDoneProgressCreate(ServerRequest):
+    token: ProgressToken
+
+    def reply(self) -> None:
```

Comment:
I'd like it if we could document shortly what replying to this actually entails/means.

Reply:
from the [spec](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#requestMessage)
> Every processed request must send a response back to the sender of the request.

nothing more